### ISSUE TITLE
Plugin not working with OpenProject 8.2

### ISSUE THREAD

**Issue by kserradell** [oliverguenther/openproject-auth_cas#3]
I've upgraded my OpenProject installation from 8.1 to 8.2.

In 8.1, this plugin was working correctly. After the upgrade, an error appears:

```
rake aborted!
Bundler::GemRequireError: There was an error while trying to load the gem 'openproject-auth_cas'.
Gem Load Error is: uninitialized constant OpenProject::Plugins::AuthPlugin
Backtrace for gem load error is:
/opt/openproject/vendor/bundle/ruby/2.5.0/bundler/gems/openproject-auth_cas-ed1d771b596f/lib/open_project/auth_cas/engine.rb:8:in `<class:Engine>'
/opt/openproject/vendor/bundle/ruby/2.5.0/bundler/gems/openproject-auth_cas-ed1d771b596f/lib/open_project/auth_cas/engine.rb:4:in `<module:AuthCas>'
/opt/openproject/vendor/bundle/ruby/2.5.0/bundler/gems/openproject-auth_cas-ed1d771b596f/lib/open_project/auth_cas/engine.rb:3:in `<module:OpenProject>'
/opt/openproject/vendor/bundle/ruby/2.5.0/bundler/gems/openproject-auth_cas-ed1d771b596f/lib/open_project/auth_cas/engine.rb:2:in `<top (required)>'
/opt/openproject/vendor/bundle/ruby/2.5.0/bundler/gems/openproject-auth_cas-ed1d771b596f/lib/open_project/auth_cas.rb:3:in `<module:AuthCas>'
/opt/openproject/vendor/bundle/ruby/2.5.0/bundler/gems/openproject-auth_cas-ed1d771b596f/lib/open_project/auth_cas.rb:2:in `<module:OpenProject>'
/opt/openproject/vendor/bundle/ruby/2.5.0/bundler/gems/openproject-auth_cas-ed1d771b596f/lib/open_project/auth_cas.rb:1:in `<top (required)>'
/opt/openproject/vendor/bundle/ruby/2.5.0/bundler/gems/openproject-auth_cas-ed1d771b596f/lib/openproject-auth_cas.rb:1:in `<top (required)>'
/opt/openproject/vendor/bundle/ruby/2.5.0/gems/bundler-1.15.2/lib/bundler/runtime.rb:82:in `require'
/opt/openproject/vendor/bundle/ruby/2.5.0/gems/bundler-1.15.2/lib/bundler/runtime.rb:82:in `block (2 levels) in require'
/opt/openproject/vendor/bundle/ruby/2.5.0/gems/bundler-1.15.2/lib/bundler/runtime.rb:77:in `each'
/opt/openproject/vendor/bundle/ruby/2.5.0/gems/bundler-1.15.2/lib/bundler/runtime.rb:77:in `block in require'
/opt/openproject/vendor/bundle/ruby/2.5.0/gems/bundler-1.15.2/lib/bundler/runtime.rb:66:in `each'
/opt/openproject/vendor/bundle/ruby/2.5.0/gems/bundler-1.15.2/lib/bundler/runtime.rb:66:in `require'
/opt/openproject/vendor/bundle/ruby/2.5.0/gems/bundler-1.15.2/lib/bundler.rb:108:in `require'
/opt/openproject/config/application.rb:70:in `<top (required)>'
/opt/openproject/Rakefile:33:in `require'
/opt/openproject/Rakefile:33:in `<top (required)>'
/opt/openproject/vendor/bundle/ruby/2.5.0/gems/rake-12.3.1/lib/rake/rake_module.rb:29:in `load'
/opt/openproject/vendor/bundle/ruby/2.5.0/gems/rake-12.3.1/lib/rake/rake_module.rb:29:in `load_rakefile'
/opt/openproject/vendor/bundle/ruby/2.5.0/gems/rake-12.3.1/lib/rake/application.rb:703:in `raw_load_rakefile'
/opt/openproject/vendor/bundle/ruby/2.5.0/gems/rake-12.3.1/lib/rake/application.rb:104:in `block in load_rakefile'
/opt/openproject/vendor/bundle/ruby/2.5.0/gems/rake-12.3.1/lib/rake/application.rb:186:in `standard_exception_handling'
/opt/openproject/vendor/bundle/ruby/2.5.0/gems/rake-12.3.1/lib/rake/application.rb:103:in `load_rakefile'
/opt/openproject/vendor/bundle/ruby/2.5.0/gems/rake-12.3.1/lib/rake/application.rb:82:in `block in run'
/opt/openproject/vendor/bundle/ruby/2.5.0/gems/rake-12.3.1/lib/rake/application.rb:186:in `standard_exception_handling'
/opt/openproject/vendor/bundle/ruby/2.5.0/gems/rake-12.3.1/lib/rake/application.rb:80:in `run'
/opt/openproject/vendor/bundle/ruby/2.5.0/gems/rake-12.3.1/exe/rake:27:in `<top (required)>'
/opt/openproject/vendor/bundle/ruby/2.5.0/bin/rake:23:in `load'
/opt/openproject/vendor/bundle/ruby/2.5.0/bin/rake:23:in `<top (required)>'
/opt/openproject/vendor/bundle/ruby/2.5.0/gems/bundler-1.15.2/lib/bundler/cli/exec.rb:74:in `load'
/opt/openproject/vendor/bundle/ruby/2.5.0/gems/bundler-1.15.2/lib/bundler/cli/exec.rb:74:in `kernel_load'
/opt/openproject/vendor/bundle/ruby/2.5.0/gems/bundler-1.15.2/lib/bundler/cli/exec.rb:27:in `run'
/opt/openproject/vendor/bundle/ruby/2.5.0/gems/bundler-1.15.2/lib/bundler/cli.rb:360:in `exec'
/opt/openproject/vendor/bundle/ruby/2.5.0/gems/bundler-1.15.2/lib/bundler/vendor/thor/lib/thor/command.rb:27:in `run'
/opt/openproject/vendor/bundle/ruby/2.5.0/gems/bundler-1.15.2/lib/bundler/vendor/thor/lib/thor/invocation.rb:126:in `invoke_command'
/opt/openproject/vendor/bundle/ruby/2.5.0/gems/bundler-1.15.2/lib/bundler/vendor/thor/lib/thor.rb:369:in `dispatch'
/opt/openproject/vendor/bundle/ruby/2.5.0/gems/bundler-1.15.2/lib/bundler/cli.rb:20:in `dispatch'
/opt/openproject/vendor/bundle/ruby/2.5.0/gems/bundler-1.15.2/lib/bundler/vendor/thor/lib/thor/base.rb:444:in `start'
/opt/openproject/vendor/bundle/ruby/2.5.0/gems/bundler-1.15.2/lib/bundler/cli.rb:10:in `start'
/opt/openproject/vendor/bundle/ruby/2.5.0/gems/bundler-1.15.2/exe/bundle:30:in `block in <top (required)>'
/opt/openproject/vendor/bundle/ruby/2.5.0/gems/bundler-1.15.2/lib/bundler/friendly_errors.rb:121:in `with_friendly_errors'
/opt/openproject/vendor/bundle/ruby/2.5.0/gems/bundler-1.15.2/exe/bundle:22:in `<top (required)>'
/opt/openproject/bin/bundle:3:in `load'
/opt/openproject/bin/bundle:3:in `<main>'
Bundler Error Backtrace:
/opt/openproject/vendor/bundle/ruby/2.5.0/gems/bundler-1.15.2/lib/bundler/runtime.rb:85:in `rescue in block (2 levels) in require'
/opt/openproject/vendor/bundle/ruby/2.5.0/gems/bundler-1.15.2/lib/bundler/runtime.rb:81:in `block (2 levels) in require'
/opt/openproject/vendor/bundle/ruby/2.5.0/gems/bundler-1.15.2/lib/bundler/runtime.rb:77:in `each'
/opt/openproject/vendor/bundle/ruby/2.5.0/gems/bundler-1.15.2/lib/bundler/runtime.rb:77:in `block in require'
/opt/openproject/vendor/bundle/ruby/2.5.0/gems/bundler-1.15.2/lib/bundler/runtime.rb:66:in `each'
/opt/openproject/vendor/bundle/ruby/2.5.0/gems/bundler-1.15.2/lib/bundler/runtime.rb:66:in `require'
/opt/openproject/vendor/bundle/ruby/2.5.0/gems/bundler-1.15.2/lib/bundler.rb:108:in `require'
/opt/openproject/config/application.rb:70:in `<top (required)>'
/opt/openproject/Rakefile:33:in `require'
/opt/openproject/Rakefile:33:in `<top (required)>'
/opt/openproject/vendor/bundle/ruby/2.5.0/gems/rake-12.3.1/exe/rake:27:in `<top (required)>'
/opt/openproject/vendor/bundle/ruby/2.5.0/gems/bundler-1.15.2/lib/bundler/cli/exec.rb:74:in `load'
/opt/openproject/vendor/bundle/ruby/2.5.0/gems/bundler-1.15.2/lib/bundler/cli/exec.rb:74:in `kernel_load'
/opt/openproject/vendor/bundle/ruby/2.5.0/gems/bundler-1.15.2/lib/bundler/cli/exec.rb:27:in `run'
/opt/openproject/vendor/bundle/ruby/2.5.0/gems/bundler-1.15.2/lib/bundler/cli.rb:360:in `exec'
/opt/openproject/vendor/bundle/ruby/2.5.0/gems/bundler-1.15.2/lib/bundler/vendor/thor/lib/thor/command.rb:27:in `run'
/opt/openproject/vendor/bundle/ruby/2.5.0/gems/bundler-1.15.2/lib/bundler/vendor/thor/lib/thor/invocation.rb:126:in `invoke_command'
/opt/openproject/vendor/bundle/ruby/2.5.0/gems/bundler-1.15.2/lib/bundler/vendor/thor/lib/thor.rb:369:in `dispatch'
/opt/openproject/vendor/bundle/ruby/2.5.0/gems/bundler-1.15.2/lib/bundler/cli.rb:20:in `dispatch'
/opt/openproject/vendor/bundle/ruby/2.5.0/gems/bundler-1.15.2/lib/bundler/vendor/thor/lib/thor/base.rb:444:in `start'
/opt/openproject/vendor/bundle/ruby/2.5.0/gems/bundler-1.15.2/lib/bundler/cli.rb:10:in `start'
/opt/openproject/vendor/bundle/ruby/2.5.0/gems/bundler-1.15.2/exe/bundle:30:in `block in <top (required)>'
/opt/openproject/vendor/bundle/ruby/2.5.0/gems/bundler-1.15.2/lib/bundler/friendly_errors.rb:121:in `with_friendly_errors'
/opt/openproject/vendor/bundle/ruby/2.5.0/gems/bundler-1.15.2/exe/bundle:22:in `<top (required)>'
/opt/openproject/bin/bundle:3:in `load'
/opt/openproject/bin/bundle:3:in `<main>'

Caused by:
NameError: uninitialized constant OpenProject::Plugins::AuthPlugin
```
I've never written a single line of Ruby so I'm quite lost. Is this plugin still maintained?

Thank you!

**Comment by kserradell** [comment]
I think the issue is related to [auth_plugins](https://github.com/opf/openproject-auth_plugins) deprecation.

**Comment by kserradell** [comment]
@oliverguenther [this commit](https://github.com/kserradell/openproject-auth_cas/commit/c5a9da6395a54aab7da14f471b00b1d666341dd5) fixes the error. I don't if it's the nice way (first line in Ruby) but CAS auth is working again in our site.

**Comment by oliverguenther** [comment]
Thanks @kserradell for the pull request :+1:  Released this as 0.1.1
